rholang/src/test/scala/rholang/interpreter/NormalizeTest.scala

```diff
@@ -0,0 +1,231 @@
+import coop.rchain.interpreter._
```

Comment:
This file is missing a package declaration:
```scala
package coop.rchain.rholang.interpreter
```

(I understand that that other files use the package `coop.rchain.interpreter`, but those should probably be changed as well)

The path to this file should mirror the package name, just like in Java:
```
./src/test/scala/coop/rchain/rholang/interpreter/NormalizeTest.scala
```

Reply:
done.